BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -69,30 +69,44 @@ private struct SearchableVaultListView: View {
 
     /// A view that displays the empty vault interface.
     @ViewBuilder private var emptyVault: some View {
-        GeometryReader { reader in
-            ScrollView {
-                VStack(spacing: 24) {
-                    vaultFilterRow
-                        .padding(.top, 16)
-
-                    Spacer()
-
-                    Text(Localizations.noItems)
-                        .multilineTextAlignment(.center)
-                        .styleGuide(.callout)
-                        .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-
-                    Button(Localizations.addAnItem) {
-                        store.send(.addItemPressed)
-                    }
-                    .buttonStyle(.tertiary())
+        VStack(spacing: 24) {
+            vaultFilterRow
+                .padding(.top, 16)
+
+            Spacer()
+
+            Image(decorative: Asset.Images.emptyVaultItems)
+                .resizable()
+                .frame(width: 100, height: 100)
+
+            VStack(spacing: 12) {
+                Text(Localizations.saveAndProtectYourData)
+                    .styleGuide(.title2, weight: .semibold)
+                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
+
+                Text(Localizations.theVaultProtectsMoreThanJustPasswordsStoreSecureLoginsIdsCardsAndNotesSecurelyHere)
+                    .styleGuide(.body)
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
+                    .padding(.horizontal, 8)
+            }
 
-                    Spacer()
+            Button {
+                store.send(.addItemPressed)
+            } label: {
+                HStack {
+                    Image(decorative: Asset.Images.plus)
+                        .resizable()
+                        .frame(width: 16, height: 16)
+                    Text(Localizations.addAnItem)
                 }
-                .padding(.horizontal, 16)
-                .frame(minHeight: reader.size.height)
             }
+            .buttonStyle(.primary(shouldFillWidth: false))
+            .clipShape(RoundedRectangle(cornerRadius: 28))
```

Comment:
🎨 I think we could leave the clipping out of this. There's going to be a ticket as part of the redesign to update all buttons to that new style and so we can build it into the button style at that point.